justfile

```diff
@@ -79,7 +79,7 @@ build-docker-images:
     scripts/build-docker-images-native
 
 # generate rust bindings for contracts
-REGEXP := "^LightClient$|^LightClientStateUpdateVK$|^FeeContract$|PlonkVerifier$|^ERC1967Proxy$|^LightClientMock$|^LightClientStateUpdateVKMock$|^PlonkVerifier2$"
+REGEXP := "^LightClient$|^LightClientStateUpdateVK$|^FeeContract$|PlonkVerifier$|^ERC1967Proxy$|^LightClientMock$|^LightClientStateUpdateVKMock$|^PlonkVerifier2$|^PermissionedStakeTable$"
```

Comment:
hey @alxiong, do we still need PlonkVerifier2?

Reply:
also we no longer need the fee contract i don't think? @sveitser

Reply:
re: PV2
we don't. we can remove them (can always fish out of the git history if needed)

Reply:
> also we no longer need the fee contract i don't think? @sveitser

We don't need fee contract anymore?

Reply:
@alysiahuggins how do the builders pay fees if we don't have the fee contract anymore?

Reply:
We are using the fee contract.